django_ninja_crudl/endpoints/partial_update.py

```
@@ -94,18 +88,36 @@
             if not self.has_object_permission(request_details):
                 return self.get_404_error(request)  # noqa: WPS220
+            self.pre_patch(request_details)
+
+            simple_fields, relational_fields = self._get_fields_to_set(
+                config.model, payload
+            )
+
+            def update() -> None:
+                nonlocal obj
+                with validating_manager(config.model):  # noqa: WPS220
+                    # TODO(phuongfi91): should we use validating_manager later as well?
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)